src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'clear') {
+      for (const key of Object.keys(state)) {
+        delete state[key];
+      }
```

Comment:
You can write it with one line
```suggestion
      Object.keys(state).forEach(key => delete state[key])
```

Reply:
I would have used map, but since it wasn't in the lessons yet, I used the old for

Reply:
In this case, the best option is forEach ?